pytorch_lightning/trainer/supporters.py

```diff
@@ -473,11 +490,12 @@ def __getstate__patch__(*_):
         return iterator
 
     @staticmethod
-    def _calc_num_batches(loaders: Any) -> Union[int, float]:
+    def _calc_num_batches(loaders: Any, mode="min_size") -> Union[int, float]:
```

Comment:
Shouldn't this be "min_size_cycle"?

Reply:
There isn't any cycle if min_size is used. I believe it is why it was chosen this way. It should be changed within another PR if we want to.